gateway_code/utils/openocd.py

```diff
@@ -30,18 +30,21 @@
 import atexit
 
 import logging
+from collections import namedtuple
 
 from gateway_code import common
 from . import subprocess_timeout
 
 LOGGER = logging.getLogger('gateway_code')
 
+Openocdargs = namedtuple("OpenOCDArgs", ['path', 'config_file', 'opts'])
```

Comment:
but rename it OpenOCDArgs (not nice but I think better)

Reply:
I had it with that case but code_check didnt like it for some reason